src/js/media-controller.js

```diff
@@ -551,6 +552,17 @@ class MediaController extends MediaContainer {
     this.enableHotkeys();
   }
 
+  get fullscreenElement() {
+    if (this.hasAttribute('fullscreen-element')) {
+      return document.getElementById(this.getAttribute('fullscreen-element'));
```

Comment:
I think this is probably good enough for the majority usecase. If someone has nested WCs, they can use the setter.

Reply:
Yeah, at the very least, it feels like something that could be a followup.